packages/tools/devtools/devtools-core/src/data-visualization/VisualTree.ts

```diff
@@ -68,6 +68,21 @@ export interface VisualNodeBase {
 	 * Consumers of this value should attempt to handle unrecognized values gracefully.
 	 */
 	nodeKind: VisualNodeKind | string;
+
+	/**
```

Comment:
This comment doesn't appear to be attached to anything. Leftover from a copy-paste?

Reply:
I think I got rid of the hanging/repeat comments